config/jobs/image-pushing/k8s-staging-csi-gen.sh

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash -e
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+output="$(dirname $0)/k8s-staging-csi.yaml"
+repos="
+csi-driver-host-path
+csi-driver-iscsi
+csi-driver-nfs
+csi-driver-smb
+csi-proxy
+csi-test
+external-attacher
+external-health-monitor
+external-provisioner
+external-resizer
+external-snapshotter
+livenessprobe
+node-driver-registrar
+"
+
+cat >"$output" <<EOF
```

Comment:
For consistency purposes I would suggest to enclose the variables in braces (ref. https://google.github.io/styleguide/shellguide.html#variable-expansion):

```suggestion
cat > "${output}" <<EOF
```